rholang/src/test/scala/coop/rchain/rholang/ProcGen.scala

```diff
@@ -20,7 +21,7 @@ object tools {
 
   def streamSingleton[T](v: T): Stream[T] = v #:: Stream.empty[T]
 
-  def nonemptyString(g: Gen[Char], size: Int): Gen[String] = Gen.nonEmptyListOf(g).map(_.mkString)
+  def nonemptyString(size: Int): Gen[String] = Gen.resize(size, Gen.alphaStr)
```

Comment:
This is going to emit empty strings as well AFAIK

Reply:
Give it a try if in doubt

Reply:
Yes, because it chooses the size from `(0, max(size, 0))`